backend/src/main/java/com/huellapositiva/domain/service/ProposalService.java

```diff
@@ -38,4 +58,51 @@ public void enrollVolunteer(String proposalId, Volunteer volunteer) {
         proposal.inscribeVolunteer(volunteer);
         proposalRepository.save(proposal);
     }
+
+    /**
+     * Fetch the ESAL of the database for get the ContactPerson, change status of the proposal and send an email with the revision of the reviser.
+     *
+     * @param proposalId : The id of the proposal to be revised.
+     * @param revisionDto : Contains the email reviser and the feedback if has it.
+     * @param proposalURI : URI the proposal to revise.
+     * @param accountId Account ID of logged user
+     */
+    public ProposalRevisionEmail proposalChangeStatusToChangedRequested(
```

Comment:
We are in the proposalService, so no need to mention it in every name, it can be deduced from the context. How about?
```
public ProposalRevisionEmail requestChanges(...) {
}
```

Reply:
It's done.